### PR TITLE
add convolutiondepthwise/crop/sigmoid/tanh in operators.md

### PR DESCRIPTION
**Pull request by cavalleria**

### CODE REVIEW

File: docs/developer-guide/operators.md [hunk 1]
```diff
@@ -9,9 +9,13 @@
 * [clip](#clip)
 * [concat](#concat)
 * [convolution](#convolution)
+* [convolutiondepthwise](#convolutiondepthwise)
+* [crop](#crop)
 * [dequantize](#dequantize)
 * [lstm](#lstm)
 * [softmax](#softmax)
+* [sigmoid](#sigmoid)
```
Comment (nihui) [inline docs/developer-guide/operators.md:17]: sort by alphabet

File: docs/developer-guide/operators.md [hunk 2]
```diff
@@ -223,3 +286,21 @@ softmax(x, axis)
...
 |1|fixbug0|int|0|hack for bug fix, should be 1|
+
+# sigmoid
+```
+y = 1 / (1 + exp(-x))
+```
+
+* one_blob_only
+* support_inplace
+
```
Comment (nihui) [inline docs/developer-guide/operators.md:297]: sort by alphabet

File: docs/developer-guide/operators.md [hunk 3]
```diff
@@ -223,3 +286,21 @@ softmax(x, axis)
...
+# tanh
+```
+y = exp(x) - exp(-x) / exp(x) + exp(-x)
```
Comment (nihui) [inline docs/developer-guide/operators.md:300]: y = tanh(x)

**Review by nihui — CHANGES_REQUESTED** [review]

File: docs/developer-guide/operators.md [hunk 4]
```diff
@@ -174,17 +178,76 @@ y = activation(x3, act_type, act_params)
 |15|pad_right|int|pad_left|
 |14|pad_top|int|pad_left|
 |16|pad_bottom|int|pad_top|
+|17|impl_type|int|0|
```
Comment (nihui) [inline docs/developer-guide/operators.md:181]: impl_type has been removed

**Review by nihui — CHANGES_REQUESTED** [review]

File: docs/developer-guide/operators.md (hunk 4 above)
Comment (cavalleria) [reply docs/developer-guide/operators.md:181]: it still in `convolution.cpp,` can it be deleted?

Comment (nihui) [reply docs/developer-guide/operators.md]: no, the impl_type in convolution implementation is just for compatibility.
new models generated by ncnnoptimize will not contain this field anymore

**Comment by nihui** [comment]
thanks

**Pull request merged**